### PR TITLE
Frontier - Add Gravity Bridge wBTC

### PR DESCRIPTION
**Pull request by ChristianBorst**
This PR replaces @jkilpatr 's PR, since it adds wBTC and moves changes from `osmosis-1/osmosis-1.assetlist.json` to `osmosis-1/osmosis-frontier.assetlist.json`

### CODE REVIEW

**Comment by JeremyParish69** [comment]
g-BTC liquidity is required frist

**Review by IbrarMakaveli — CHANGES_REQUESTED** [review]
Missing pool for WBTC, USDT 
https://github.com/osmosis-labs/assetlists/pull/149

**Comment by ChristianBorst** [comment]
Thanks @JeremyParish69  and @IbrarMakaveli for helping out with this one, and sorry for the delay on getting this PR in working order (lots of delays in getting the WBTC pool created).  Big thanks @IbrarMakaveli for making #149, I have added the WBTC asset and images here.

Here is the WBTC pool: https://frontier.osmosis.zone/pool/694 with the Gravity Bridge bridged WBTC asset.

IBC denom for WBTC
```
$ osmosisd q ibc-transfer denom-trace C9B0D48FD2C5B91135F118FF2484551888966590D7BDC20F6A87308DBA670796
denom_trace:
  base_denom: gravity0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599
  path: transfer/channel-144
```
Removing "gravity" from the base_denom and plugging that into etherscan shows the bridged asset: https://etherscan.io/address/0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599

osmosis-labs/osmosis-frontend Frontier PR coming soon.

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 1]
```diff
@@ -1517,6 +1517,36 @@
...
+          ]
+        },
+        {
+          "denom": "btc",
+          "exponent": 8
+        }
+      ],
+      "base": "ibc/C9B0D48FD2C5B91135F118FF2484551888966590D7BDC20F6A87308DBA670796",
+      "name": "Wrapped Bitcoin",
+      "display": "Wrapped BTC",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1537]: 'display' property is defining which denomination to show. Please change to 'btc'
`"display": "btc"`

Comment (ChristianBorst) [reply osmosis-1/osmosis-frontier.assetlist.json]: Fixed!

**Comment by ChristianBorst** [comment]
@JeremyParish69 I have resolved your comment.

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 2]
```diff
@@ -1517,6 +1517,36 @@
...
+      "denom_units": [
+        {
+          "denom": "ibc/C9B0D48FD2C5B91135F118FF2484551888966590D7BDC20F6A87308DBA670796",
+          "exponent": 0,
+          "aliases": [
+            "satoshi"
+          ]
+        },
+        {
+          "denom": "btc",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1531]: please use 'gwbtc'

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 3]
```diff
@@ -1517,6 +1517,36 @@
...
+          "exponent": 8
+        }
+      ],
+      "base": "ibc/C9B0D48FD2C5B91135F118FF2484551888966590D7BDC20F6A87308DBA670796",
+      "name": "Wrapped Bitcoin",
+      "display": "btc",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1537]: please use 'gwbtc'

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 4]
```diff
@@ -1517,6 +1517,36 @@
...
+      "ibc": {
+        "source_channel": "channel-10",
+        "dst_channel": "channel-144",
+        "source_denom": "gravity0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/gwbtc.png",
+        "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/gwbtc.svg"
+      },
+      "coingecko_id": "bitcoin"
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1548]: please use 'wrapped-bitcoin'

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 5]
```diff
@@ -1517,6 +1517,36 @@
...
+      "name": "Wrapped Bitcoin",
+      "display": "btc",
+      "symbol": "gWBTC",
+      "ibc": {
+        "source_channel": "channel-10",
+        "dst_channel": "channel-144",
+        "source_denom": "gravity0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/gwbtc.png",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1545]: the name and url are fine, but please use this image instead:
https://raw.githubusercontent.com/cosmos/chain-registry/master/gravitybridge/images/gwbtc.png

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 6]
```diff
@@ -1517,6 +1517,36 @@
...
+        "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/gwbtc.svg"
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1546]: for now don't need this svg

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 7]
```diff
@@ -1517,6 +1517,36 @@
...
+        {
+          "denom": "ibc/C9B0D48FD2C5B91135F118FF2484551888966590D7BDC20F6A87308DBA670796",
+          "exponent": 0,
+          "aliases": [
+            "satoshi"
+          ]
+        },
+        {
+          "denom": "btc",
+          "exponent": 8
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:1532]: after switching to wtbc, is exponent 8 still accurate?

Comment (ChristianBorst) [reply osmosis-1/osmosis-frontier.assetlist.json:1532]: Yes, thankfully they set the exponent correctly when deploying the ERC20 contract: https://etherscan.io/token/0x2260fac5e5542a773aa44fbcfedf7c193bc2c599

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 2 above)
Comment (ChristianBorst) [reply osmosis-1/osmosis-frontier.assetlist.json]: Fixed

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 3 above)
Comment (ChristianBorst) [reply osmosis-1/osmosis-frontier.assetlist.json]: Fixed

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 4 above)
Comment (ChristianBorst) [reply osmosis-1/osmosis-frontier.assetlist.json]: Fixed

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 5 above)
Comment (ChristianBorst) [reply osmosis-1/osmosis-frontier.assetlist.json]: Replaced the image

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 6 above)
Comment (ChristianBorst) [reply osmosis-1/osmosis-frontier.assetlist.json]: Removed the svg entry in logo_URIs and the file in images/

**Comment by ChristianBorst** [comment]
@JeremyParish69 I have resolved all of your comments.

**Comment by sunnya97** [comment]
@JeremyParish69 is this good to merge?

**Pull request merged**